.github/composite_actions/log_cw_metric/action.yaml

```diff
@@ -0,0 +1,61 @@
+name: Log CW Metric
+description: Test to log a CW metric
+inputs:
+  # For getting failing step
+  job-status:
+    description: Used to determine if we track success or failure.
+    required: true
+  job-identifier:
```

Comment:
github does not return the current job id.  While you can get current run id, when requesting run information you get all the jobs for that run.  Thus to differentiate an individual job of that run, I needed to have this parameter for finding the particular job.